tests/core.test.ts

```diff
@@ -373,7 +373,7 @@ describe(`Core`, () => {
     it(`should extract strings from complex options (=)`, () => {
         const cli = makeCli([
             b => {
-                b.addOption({names: [`--foo`], arity: 1});
+                b.addOption({names: [`--foo`], arity: 0});
```

Comment:
The last question and I promise I'll leave you alone 😄: Shouldn't the arity be kept at `1` here, since here we're simulating a string option without `tolerateBoolean`?

Reply:
It's fine :)
Yeah I wanted to revert that change but forgot. Thanks!